packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -121,9 +124,13 @@ const withInfiniteScroll = (
     componentDidMount() {
       this.fetchItems({
         elementsPerScroll: this.props.initiallyLoadedElements,
-      }).then(newState => {
-        this.setState(newState);
-      });
+      })
+        .then(newState => {
+          this.setState(newState);
+        })
+        .catch(() => {
```

Comment:
```suggestion
        .catch(this.handleFetchItemsFailed);
```
Bit shorter and has the same effect.